src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1287,6 +1287,15 @@
   <data name="PreferStreamAsyncMemoryOverloadsMessage" xml:space="preserve">
     <value>Change the '{0}' method call to the overload that receives a '{1}'.</value>
   </data>
+  <data name="ForwardCancellationTokenToAsyncMethodsDescription" xml:space="preserve">
+    <value>Description - Forward cancellation token to async methods</value>
+  </data>
+  <data name="ForwardCancellationTokenToAsyncMethodsMessage" xml:space="preserve">
+    <value>Message - Forward cancellation token to async methods</value>
+  </data>
+  <data name="ForwardCancellationTokenToAsyncMethodsTitle" xml:space="preserve">
+    <value>Title - Forward cancellation token to async methods</value>
+  </data>
```

Comment:
Why the "Description - " prefix, and similar on the other two?

Reply:
I listed this in the PR description among the pending things to address.

Reply:
Fixed. Please let me know if you think another wording would be better.